src/emit/instructions.rs

```diff
@@ -595,7 +595,7 @@ pub(super) fn process_instruction<'a, T: TargetRuntime<'a> + ?Sized>(
                 }
             }
 
-            let ret = target.builtin_function(bin, func, &parms, ns);
+            let ret = target.builtin_function(bin, function, func, &parms, ns);
```

Comment:
I believe the bad naming here might have led to a confusion. What you think of giving better names to `func` and `function`?

Reply:
I've renamed them